src/test/resources/testDataFiles/LogoutTestData.json

```diff
@@ -0,0 +1,4 @@
+{
+  "UserEMail" : "lenesah401@mcuma.com",
```

Comment:
Can you add something related to automation like auto or bot like we are doing on other cases?